### DISCUSSION TITLE
Ideas para comandos de bot de discord

### DISCUSSION THREAD

**Discussion by Poseidon2579** [community/community#64534] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Solo dígame ideas

**Comment by TostadoraDev** [comment]
Puedes usar por ejemplo sistema de leveling o algo dinamico

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
